pytri/js/GraphLayer.js

```diff
@@ -100,6 +117,6 @@ class ColorGraphLayer extends Layer {
             })
         );
         self.children.push(edges);
-        scene.add(edges);     
+        scene.add(edges);          
```

Comment:
@hpcowley ?

in VSCode, set `  "editor.trimAutoWhitespace": true` in your settings to to prevent randomly adding whitespace like this

Reply:
don't worry about it in this PR, whoever opens the file next can fix :) probably that'll be you anyway 😎

Reply:
Ack! Sorry about that and thanks for the tip :)